pkg/addon-operator/bootstrap.go

```diff
@@ -14,7 +14,7 @@ import (
 
 // Bootstrap inits all dependencies for a full-fledged AddonOperator instance.
 func (op *AddonOperator) bootstrap() error {
-	log.Infof(sh_app.AppStartMessage)
+	log.Info(sh_app.AppStartMessage)
```

Comment:
All edits to Infof -> Info or Errorf -> Error are made to fix golangci-lint errors that occur when running locally.